src/middlewared/middlewared/plugins/crypto.py

```diff
@@ -2660,7 +2667,7 @@ def set_defaults(attr):
                 ]
             }
         """
-        ca = await self._get_instance(id)
+        ca = await self._get_instance(id)  # noqa: is assigned to but never used
```

Comment:
We should not assign to ca but keep the get instance call as it ensures that a ca of that id exists